package.json

```diff
@@ -21,5 +24,42 @@
   "bugs": {
     "url": "https://github.com/ZbayApp/identity/issues"
   },
-  "homepage": "https://github.com/ZbayApp/identity#readme"
+  "homepage": "https://github.com/ZbayApp/identity#readme",
+  "files": [
+    "/lib/**/*"
+  ],
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest",
+      "^.+\\.jsx?$": "babel-jest"
+    },
+    "testPathIgnorePatterns": [
+      "/node_modules/"
+    ],
+    "transformIgnorePatterns": [
+      "/node_modules/"
+    ],
+    "testRegex": ".*\\.test\\.(t|j)s$",
+    "testEnvironment": "node"
+  },
+  "dependencies": {
+    "pkijs": "^2.1.95",
+    "@peculiar/webcrypto": "^1.1.7",
+    "node-webcrypto-ossl": "^2.1.2"
```

Comment:
This is only used in tests, should be dev dependency

Reply:
Moved